app/controllers/api/v2/autosign_controller.rb

```diff
@@ -19,6 +39,10 @@ def setup_proxy
       def allowed_nested_id
         %w(smart_proxy_id)
       end
+
+      def handle_proxy_error exception
+        render :status => 500, :json => { "error": exception.message }
```

Comment:
this seems to break on Ruby 2.0 and 2.1, i.e. http://ci.theforeman.org/job/packaging_build_deb_coreproject/arch=x86,label=debian,os=jessie/4129/console